editor/store.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { applyMiddleware, createStore } from 'redux';
+import refx from 'refx';
+import multi from 'redux-multi';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import effects from './effects';
+import reducer from './reducer';
+import { getPreferences } from './selectors';
+
+/**
+ * Module constants
+ */
+const GUTENBERG_PREFERENCES_KEY = 'GUTENBERG_PREFERENCES';
+
+/**
+ * Loads the initial preferences and saves them to the local storage once changed
+ * @param {Object} store Redux Store
+ */
+function setupStorePersistence( store ) {
```

Comment:
Absolutely, thanks for mentioning this. I extracted the behaviour into a generic simple store enhancer allowing to persist and autohydrate a specific reducer key.

It could be easily extended later (or replaced by redux persist if needed).